ansible/main.yml

```diff
@@ -1,4 +1,41 @@
 ---
+
+- name: Project Bootstrap
+  hosts: all
+  tasks:
+    - name: Get User Token
+      uri:
+        url: "https://{{ clusterhost }}/oauth/authorize?response_type=token&client_id=openshift-challenging-client"
+        user: "{{ openshift_username }}"
+        password: "{{ openshift_password }}"
+        validate_certs: False
+        force_basic_auth: True
+      register: login
```

Comment:
I think this would not work on clusters whose identity providers don't support challenge authentication... and in any case we probably want to support pre-authentication...

Reply:
@codificat ok I will add a when clause if token is already defined.